app/src/main/java/com/pnow/rick_and_morty_list/app/ui/fragment/CharactersListFragment.kt

```diff
@@ -8,14 +8,13 @@ import androidx.fragment.app.Fragment
 import androidx.fragment.app.viewModels
 import androidx.lifecycle.lifecycleScope
 import androidx.navigation.findNavController
-import androidx.paging.PagingData
 import androidx.recyclerview.widget.LinearLayoutManager
 import com.pnow.rick_and_morty_list.R
 import com.pnow.rick_and_morty_list.app.ui.adapter.CharacterListAdapter
-import com.pnow.rick_and_morty_list.app.ui.model.CharacterUIModel
 import com.pnow.rick_and_morty_list.app.ui.viewmodel.CharacterViewModel
 import com.pnow.rick_and_morty_list.databinding.FragmentCharactersListBinding
 import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.flow.collectLatest
 import kotlinx.coroutines.launch
 
 
```

Comment:
w lini 30 masz nullowalny view

obczaj czy to nie rozwiąże: `binding.root.findNavController()` albo do zmiennej albo po prostu

na 100% nie pamiętam już jak to się robiło więc nie jestem pewny czy zadzaila

Reply:
nie mogę comentować kodu który nie jest jakoś tam z związany z PR (wtf) 

linia 34 czy nie sensownej żeby to było w fragmencie który odbiera dane? pomyśl że nie znasz projektu wchodzisz do fragmentu i widzisz, że ma jakieś takie rzeczy, które mają (spoko nazwę a nie taką jak tu) i wiesz co się dzieje do razu ; )